### ISSUE TITLE
Support Perceptual Flow Matching (PFM) for Few-Step Generation and Distillation

### ISSUE THREAD

**Issue by ZhaoChuyang** [modelscope/DiffSynth-Studio#1518]
Hi! We recently published our work, Perceptual Flow Matching for Few-Step Generative Modeling: https://arxiv.org/abs/2607.03524.

In this work, we show that simply replacing the standard MSE loss with a perceptual loss during flow matching training can produce a strong few-step generator with only a few hundred training steps. We have validated the method on SD3, Qwen-Image-Edit, and Wan. The Qwen-Image-Edit and Wan experiments were both implemented based on DiffSynth-Studio.

I was wondering whether you would be interested in integrating PFM into DiffSynth-Studio. The method is highly compatible with the current SFT training pipeline, as it only requires replacing the training loss while leaving the rest of the framework unchanged.

If this sounds interesting, I'd be happy to help integrate our existing implementation and submit a PR for the project.

**Comment by shoutOutYangJie** [comment]
Yes, please integrate your code, and also pulish your code in the independent  repo

**Comment by dongdk** [comment]
@ZhaoChuyang when will u support it?

**Comment by igodogi** [comment]
P0跟进

**Comment by dongdk** [comment]
any update？

**Comment by dongdk** [comment]
any update？
